src/kajal/ScannerExamples/Assignment14p3.java

```diff
@@ -0,0 +1,151 @@
+/*
+	  Assignment 14: Shop using user input (using scanner Class)
+	  Added new logic as if user input only 3 quantities not all then we can pass single parameter to see expected output and
+	   avoided passing 0 in parameter
+
+ */
+package kajal.ScannerExamples;
+
+import java.util.Scanner;
+
+public class Assignment14p3 {
+
+		int maggieQuantity=50, dosaQuantity=43, pouchesQuantity=39, panipuriQuantity=43, masalaQuantity=73;
+
+		void requestedQMaggie(int maggieQ) {
+			if(maggieQ<=50 && maggieQ!=0 ) {
```

Comment:
this 50, 39, and 43 etc should not be hard coded @KajalKirdat 

take reference of below code
Astha - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/584/files
Krishna S - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/589/files